### ISSUE TITLE
Not working on High Sierra

### ISSUE THREAD

**Issue by idanhaviv** [dteoh/SlowQuitApps#40]
Installed using Homebrew, gave it the accessibility permissions, and it's running but the cmd+q doesn't have any delay. 
Running 'defaults read com.dteoh.SlowQuitApps' returns 'Domain com.dteoh.SlowQuitApps does not exist'

**Comment by glimpsed** [comment]
Had the same problem on macOS Mojave, reinstalled the app, gave it permission to control the computer via Security & Privacy > Privacy > Accessibility, and now it works.

**Comment by dteoh** [comment]
Can you give v0.6.0 a go. I was not sure why it could still break after giving permissions, but this newer version uses newer APIs which would hopefully give less problems.

**Comment by Saratxe** [comment]
Hi!
Updated from V0.50 to V0.60 and it doesn't work anymore. CMD+Q shows the delay just for a blink of a second but doesn't quit the app. Have to press CMD+Q several times before it works like before. Went back to V0.50 and it works perfectly. I'm using an iMac with High Sierra 10.13.6.

**Comment by Phuker** [comment]
Hi!
I updated from V0.50 to V0.60 and there's some problem. The overlay looks overflow (more than 360 degree), and sometimes it fails to quit the app.
Macbook Pro 2018 15-inch with touch bar, macOS High Sierra 10.13.6.
Thanks.

**Comment by dteoh** [comment]
I don't have a High Sierra install running directly on hardware, so I tested the app in a fresh VM install. The app works as expected.

I don't really know why this issue occurs.

On Mojave, I upgraded my work computer and ran into the problem described by @Saratxe. I had to:

* quit the app
* remove the existing accessibility permissions
* delete the app
* install the new version of the app
* open the app with right click, grant permission to open because the app is not signed
* grant accessibility permissions
* reboot

... and then it works again.

**Comment by ErikMHummel** [comment]
I had this problem and just needed to do `killall SlowQuitApps` And reopen and all was good.

**Comment by Phuker** [comment]
I uninstall & install & rebooted as the instructions above, but the problem sill exists.
But it seems that I know some clue now. I set delay time using this command:
`defaults write com.dteoh.SlowQuitApps delay -int 225`
It works fine in 0.50 but not good after I upgraded to 0.60. I tried 1000ms in version 0.60, and the overlay looks normal now.
This is how it looks with delay 225ms in version 0.60: 
![slowquitapps](https://user-images.githubusercontent.com/11596611/46951196-de2fc580-d0b9-11e8-9ede-6de8df82ddcc.gif)
